packages/sozluk-content/terms/stlc.mdx

```diff
@@ -3,6 +3,7 @@ title: STLC - Software Testing Life Cycle
 tags:
   - qa
   - sdlc
+  - testing
 ---
 
-STLC, yazılım testlerinin planlama, hazırlık, yürütme, raporlama ve kapanış aşamalarını içeren bir süreçtir. Amaç yazılımın doğru ve eksiksiz bir şekilde test edilmesini sağlamak, hataları erken tespit ederek kaliteyi artırmaktır. STLC, yazılım testlerini yönetmek ve süreci etkin bir şekilde ilerletmek için sistemli bir yaklaşım sunar.
+Yazılım testlerini yönetmek ve süreci etkin bir şekilde ilerletmek için planlama, hazırlık, yürütme, raporlama ve kapanış aşamalarını sistemli test sürecidir. Amaç yazılımın doğru ve eksiksiz bir şekilde test edilmesini sağlamak, hataları erken tespit ederek kaliteyi artırmaktır.
```

Comment:
```suggestion
Yazılım testlerini yönetmek ve süreci etkin bir şekilde ilerletmek için planlama, hazırlık, yürütme, raporlama ve kapanış aşamalarını içeren sistemli test sürecidir. Amaç yazılımın doğru ve eksiksiz bir şekilde test edilmesini sağlamak, hataları erken tespit ederek kaliteyi artırmaktır.
```

Reply:
@igdiaysu something like this

Reply:
😱

thanks ✨

Reply:
You can click to commit suggestion and send another commit to this PR in this way